src/custom/api/coingecko/index.ts

```diff
@@ -62,6 +62,7 @@ function _get(chainId: ChainId, url: string): Promise<Response> {
 export interface CoinGeckoUsdPriceParams {
   chainId: ChainId
   tokenAddress: string
+  isNative: boolean
```

Comment:
I wonder why you needed to add a `isNative` if you ended up implementing by adding the WETH address for Ether.

I would understand if its because you would use the other endpoint for native currency. But you used WETH address.